base/setup/usetup/lang/uk-UA.h

```diff
@@ -3,7 +3,7 @@
 // To revert conversion, please execute "code7bit -r <file>".
 /*
  *      translated by Artem Reznikov, Igor Paliychuk, 2010
- *      http://www.reactos.org/uk/
+ *      https://reactos.org/uk/
```

Comment:
Broken URL

Reply:
fixed

Reply:
Still broken in comment

```suggestion
 *      https://reactos.org/
```

Reply:
I fixed that 2nd one as well.